### PR TITLE
LEA-22 - Example integrating with GitHub

### PR DESCRIPTION
**Pull request by JustinGuedes**
* Create new-file to show as an example
* Update home page with title change
* Update types page with less content and added new test function

### CODE REVIEW

File: web/src/pages/lesson1/types/example.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export function test() {}
```
Comment (ConorBailey) [inline web/src/pages/lesson1/types/example.ts]: Please add functionality to this function.

File: web/src/pages/index.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { ReactElement } from "react";
 export default function Home(): ReactElement {
   return (
     <div>
-      <Heading1>Learning Code</Heading1>
+      <Heading1>Learning Code 2.0</Heading1>
```
Comment (Steobeast) [inline web/src/pages/index.tsx]: No need to put the 2.0 in

**Review by ConorBailey — CHANGES_REQUESTED** [review]

File: web/src/pages/lesson1/types/index.tsx [hunk 3]
```diff
@@ -3,13 +3,7 @@ import { ReactElement } from "react";
...
   return (
     <div>
-      Conor complete this page based off this&nbsp;
-      <a
-        target="_blank"
-        href="https://github.com/JustinGuedes/learn-coding/blob/main/lesson1/types.md"
-      >
-        page
-      </a>
+      Conor failed the assignment
```
Comment (Tatenda-D-Mawoneke) [inline web/src/pages/lesson1/types/index.tsx:6]: Why did you remove this?

File: web/src/pages/lesson1/types/example.ts (hunk 1 above)
Comment (Steobeast) [inline web/src/pages/lesson1/types/example.ts]: Function doesnt have a use, consider moving it?

**Review by Tatenda-D-Mawoneke — CHANGES_REQUESTED** [review]
Check out my comments

**Review by Steobeast — COMMENTED** [review]
Good job man

File: web/src/pages/lesson1/types/example.ts (hunk 1 above)
Comment (JustinGuedes) [reply web/src/pages/lesson1/types/example.ts]: Will do!

File: web/src/pages/lesson1/types/index.tsx (hunk 3 above)
Comment (JustinGuedes) [reply web/src/pages/lesson1/types/index.tsx:6]: Because Conor didn't know what he was doing

File: web/src/pages/lesson1/types/example.ts (hunk 1 above)
Comment (JustinGuedes) [reply web/src/pages/lesson1/types/example.ts]: I'll be adding functionality to it soon

File: web/src/pages/lesson1/types/index.tsx (hunk 3 above)
Comment (Tatenda-D-Mawoneke) [reply web/src/pages/lesson1/types/index.tsx:6]: Checks out. 
image

**Review by Tatenda-D-Mawoneke — DISMISSED** [review]

**Review by ConorBailey — APPROVED** [review]

File: web/src/pages/index.tsx [hunk 4]
```diff
@@ -20,6 +20,7 @@ export default function Home(): ReactElement {
       <div>
         <ul>
           <li>Git</li>
+          <li>Example</li>
```
Comment (Steobeast) [inline web/src/pages/index.tsx:23]: Conor still eats dick but its all good

**Review by Tatenda-D-Mawoneke — APPROVED** [review]
image

**Review by Steobeast — APPROVED** [review]
Happy

**Review by Steobeast — APPROVED** [review]
